### ISSUE TITLE
How to evaluate the base training performance?

### ISSUE THREAD

**Issue by Jesse2hong** [ZhangGongjie/Meta-DETR#69]
Hi there! thanks for sharing the great work!
But I have a problem in evaluating the base training performance.
I downloaded the pre_trained weights that you released, and conducted  inference without any fine-tuning. Follows are my inference 
command and the results on base classes. I don't know why the performance is far worse than that you released.  Could you please give me some clue?

python -u main.py \
--dataset_file coco_base \
--backbone resnet101 \
--num_feature_levels 1 \
--enc_layers 6 \
--dec_layers 6 \
--hidden_dim 256 \
--num_queries 300 \
--batch_size 2 \
--category_codes_cls_loss \
--resume path/to/checkpoint.pth \
--fewshot_finetune \
--fewshot_seed 1 \
--num_shots 10 \
--eval \
2>&1 | tee ./log_inference.txt 

 * Base Categories:
Accumulating evaluation results...
DONE (t=12.80s).
IoU metric: bbox
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.079
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.108
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.087
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.043
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.096
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.118
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.271
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.421
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.421
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.184
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.476
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.677

**Comment by xiaozhang-ok** [comment]
> Hi there! thanks for sharing the great work! But I have a problem in evaluating the base training performance. I downloaded the pre_trained weights that you released, and conducted inference without any fine-tuning. Follows are my inference command and the results on base classes. I don't know why the performance is far worse than that you released. Could you please give me some clue?
> 
> python -u main.py --dataset_file coco_base --backbone resnet101 --num_feature_levels 1 --enc_layers 6 --dec_layers 6 --hidden_dim 256 --num_queries 300 --batch_size 2 --category_codes_cls_loss --resume path/to/checkpoint.pth --fewshot_finetune --fewshot_seed 1 --num_shots 10 --eval 2>&1 | tee ./log_inference.txt
> 
> * Base Categories:
>   Accumulating evaluation results...
>   DONE (t=12.80s).
>   IoU metric: bbox
>   Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.079
>   Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.108
>   Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.087
>   Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.043
>   Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.096
>   Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.118
>   Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.271
>   Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.421
>   Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.421
>   Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.184
>   Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.476
>   Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.677

Hi, I hope you're doing well! I'm currently working with Meta-DETR and I'm trying to fine-tune the model for my specific dataset. Would you be willing to share some guidance on how to approach this fine-tuning process? I would really appreciate your help!
